tests/test_jjb.py

```diff
@@ -3,6 +3,7 @@
 import os
 import tempfile
 from jenkins_jobs.cli.entry import JenkinsJobs
+from tests.helpers import compare_jjb_output
```

Comment:
I'd suggest replacing it with a relative import, like `from .helpers import...`, to avoid potential future conflict with a 3rd party package called `tests`.